### PR TITLE
added a link to 'checking the results' for clarity

### PR DESCRIPTION
**Pull request by Andre-Rb**
Hi, 
I was following the making plugin tut', but I had some troubles getting how to make plugins work, until I re-read carefully and figured it was explained after the second plugin example.

So yeah, I guess it's clearer ? 

Thanks in advance, first PR BTW, I hope it's done correctly :)

### CODE REVIEW

File: tutorials/plugins/editor/making_plugins.rst [hunk 1]
```diff
@@ -139,7 +139,9 @@ dialog. For that, change the ``custom_node.gd`` script to the following::
         remove_custom_type("MyButton")
 
 With that done, the plugin should already be available in the plugin list at
-Project Settings. So activate it and try to add a new node to see the result:
+Project Settings, so activate it as explained in `Checking the results`_. 
+
+Then try it out by adding your new node :
```
Comment (cbscribe) [inline tutorials/plugins/editor/making_plugins.rst]: Remove the space before the colon

**Comment by mhilbrunner** [comment]
Can you update this to remove the space? If you need help with this, feel free to join us in the #documentation channel on the Godot Engine Discord or in #godotengine-devel on Freenode IRC.

**Comment by Andre-Rb** [comment]
My Bad, I thought it was done when I saw @cbscribe comment :)
Told you guys it was my first :D

**Comment by cbscribe** [comment]
Thanks!

**Pull request merged**